src/event/event.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from 'src/prisma/prisma.service';
+
+@Injectable()
+export class EventService {
+  constructor(private prismaService: PrismaService) {}
+
+  async activateEvent(eventId: number) {
+    const event = this.prismaService.event.update({
+      data: { is_active: true },
+      where: { id: Number(eventId) }, // bruh
```

Comment:
try daw use ug dto for update para didto ang casting. pero okay na man ni, merge na nako

Reply:
if gusto nimo itry with dto para mag implicit casting, ingna lang ko kay di sa nako imerge. pero if di na, kay imerge na nako ni. just lemme know @p0tatoes

Reply:
Noted. I-try nako ug refactor pra mag use ug dto for the `eventId`.